site/docsource/Build-system-support.adoc

```diff
@@ -13,17 +13,19 @@ files with the extension `.cmj` which are later used for cross module
 inlining, arity inference and other information.
 ======
 
-=== Build with bsb 
+=== BuckleScript build system: `bsb`
 
-BuckleScript proivdes a native build tool using https://github.com/ninja-build/ninja/releases[ninja], 
-it is designed for fast feedback loop (typically 100ms feedback loop). 
+BuckleScript proivdes a native build tool on top of Google's  https://github.com/ninja-build/ninja/releases[ninja-build], 
```

Comment:
thanks! there are two spaces after Google, not sure why it is not displayed here